packages/worker/src/notifier.ts

```diff
@@ -0,0 +1,68 @@
+
+import type { Config, Plugin } from '@bugsnag/core'
+import Client from '@bugsnag/core/client'
+import { schema as coreSchema } from '@bugsnag/core/config'
+import delivery from '@bugsnag/delivery-fetch'
+
+const name = 'Bugsnag JavaScript'
+const version = '__VERSION__'
```

Comment:
The translation from this to the actual version is done by [`browserify-versionify`](https://www.npmjs.com/package/browserify-versionify) here: https://github.com/bugsnag/bugsnag-js/blob/8d607ca4fd3fc1470207fdadca7aff78573a5542/bin/bundle#L16

So we'd need a webpack equivalent for replacing this when it's built